packages/CopyInput/src/CopyInput.js

```diff
@@ -100,18 +112,25 @@ const propTypes = {
   isReadOnly: PropTypes.bool,
   /** Default value for the input */
   value: PropTypes.string,
+  /** If a plain text version of the value will be rendered */
+  hasValueContainer: PropTypes.bool,
+  /** If the value will be rendered in an Input component or hidden */
+  hasInputContainer: PropTypes.bool,
```

Comment:
Oh, minor point, but it's nice if we can keep the `propTypes` and `defaultProps` in alphabetic order 🙏